x-pack/functionbeat/function/beater/functionbeat.go

```diff
@@ -31,6 +33,7 @@ var (
 		"logstash",
 		"console", // for local debugging
 	}
+	metrics = monitoring.Default.NewRegistry("functionbeat")
```

Comment:
libbeat declares that 'stat' registry for telemtry data. If it is used we will have the monitoring backend report metrics and telemetry information every now and so often.

Reply:
I've only found a registry named "state". Is that what you mean?
https://github.com/elastic/beats/blob/master/libbeat/cmd/instance/beat.go#L338

Reply:
I think @kvch is already using the `stats` registry here, by virtue of using `monitoring.Default`. See https://github.com/ycombinator/beats/blob/f35ffdca261d49c83a2a8fcbd40b2cb107c95b43/libbeat/monitoring/monitoring.go#L50-L55.

Reply:
I mean "state". The 'stats' registry is for metrics, the 'state' registry is for telemetry.